sig/lib/_test/Steepfile

```diff
@@ -0,0 +1,10 @@
+D = Steep::Diagnostic
+
+target :test do
+  check "."
+  signature "../sig"
+
+  library "jct", "date"
```

Comment:
It looks confusing.

First, `../sig` doesn't exist. It points `sig/lib/_test/../sig` (== `sig/lib/sig`). So `signature "../sig"` does nothing.


Steep loads the` sig/` directory without `signature` because of `library "jct"`. `library` method loads RBSs from installed gems. In this case, `jct` gem is installed by Bundler, so it is loaded.


I think we can just remove `signature "../sig"` from this code.

Reply:
Thanks review.
I have removed it in this commit ->  https://github.com/moneyforward/jct/pull/16/commits/ca3c187415364513d1b42b49466a2abf81f7eac0